supply/lib/supply/client.rb

```diff
@@ -298,7 +298,7 @@ def track_version_codes(track)
           current_edit.id,
           track
         )
-        return result.version_codes
+        return result.version_codes.nil? ? [] : result.version_codes
```

Comment:
This can be `result.version_codes || []`

Reply:
I agree with this ^ Can you make this change? I will then click the merge button 😊